eradiate/thermoprops/util.py

```diff
@@ -32,17 +344,17 @@ def water_vapor_saturation_pressure(t):
     .. note::
         Valid for pressures larger than the triple point pressure (~611 Pa).
 
-    Parameter ``t`` (float):
+    Parameter ``t`` (:class:`float`):
```

Comment:
Not required (honestly, nobody looks up the Python reference for the `float` type).